basic/src/test/java/org/jboss/hal/testsuite/test/configuration/webservices/configuration/handlerchain/WebServicesClientConfigurationHandlerChainTestCase.java

```diff
@@ -80,24 +83,26 @@ public void addPreHandlerChainDuplicateShowsError() throws Exception {
     }
 
     @Test
-    public void addPreHandlerChainWithoutNameShowsError() throws Exception {
-        navigateToClientConfigurationHandlerChainTab(clientConfigurationName, PRE_HANDLER_CHAIN_TAB_LABEL);
+    public void testAddPreHandlerChainWithoutNameShowsError() throws Exception {
+        page.navigate();
+        page.switchToClientConfigurationHandlerChainTab(clientConfigurationName, PRE_HANDLER_CHAIN_TAB_LABEL);
         page.getResourceManager()
                 .addResource(WebServicesHandlerChainWizard.class)
                 .saveWithState().assertWindowOpen();
         Assert.assertTrue("Validation error regarding missing pre handler name should be visible", page.getWindowFragment().isErrorShownInForm());
     }
 
     @Test
-    public void editPreHandlerChainProtocolBindings() throws Exception {
+    public void testEditPreHandlerChainProtocolBindings() throws Exception {
```

Comment:
Adding `test` prefix to `edit*` test methods it's not necessary, since there are only few methods with similar prefix in `ConfigFragment` class.